source/common/protobuf/protobuf.h

```diff
@@ -7,6 +7,7 @@
 #include "google/protobuf/any.pb.h"
 #include "google/protobuf/descriptor.h"
 #include "google/protobuf/descriptor.pb.h"
+#include "google/protobuf/descriptor_database.h"
```

Comment:
not used?

Reply:
It's intentional, I use the `DescriptorPoolDatabase` in transcoder, but I cannot include this directly, as the linter says "unexpected direct dependency on google.protobuf, use the definitions in common/protobuf/protobuf.h instead."

Reply:
Ah OK that's fine.